quill/test/HandlerCollectionTest.cpp

```diff
@@ -54,16 +54,20 @@ TEST_CASE("create_get")
 
 #if defined(_WIN32)
   // Create a file handler
-  StreamHandler* filehandler = hc.file_handler(L"create_get_file_handler", "w");
+  StreamHandler* filehandler =
+    hc.create_handler<FileHandler>(L"create_get_file_handler", "w", FilenameAppend::None);
 
   // Request the same file handler
-  StreamHandler* filehandler_2 = hc.file_handler(L"create_get_file_handler");
+  StreamHandler* filehandler_2 =
+    hc.create_handler<FileHandler>("create_get_file_handler", "a", FilenameAppend::None);
```

Comment:
@odygrd Did you forget the `L`?
```suggestion
    hc.create_handler<FileHandler>(L"create_get_file_handler", "a", FilenameAppend::None);
```